risc0/zkvm/prove/make-id/src/main.rs

```diff
@@ -12,12 +12,21 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//! Creates RISC Zero MethodIDs from ELF files
+//!
+//! A simple command line tool for creating
+//! [MethodID](risc0_zkvm::host::MethodId)s from RISC-V ELF files. It is
+//! intended for basic, low-level use cases. In particular, users looking to
+//! create MethodIDs for Rust code are advised to instead use our zkVM: visit
+//! our [Risc Zero Rust Starter repository](https://github.com/risc0/risc0-rust-starter)
+//! for help getting started.
```

Comment:
I don't think that binary targets will ever show up in `docs.rs` or rustdocs in general. But it's fine to have comments here describing the binary, perhaps just as normal non-rustdoc comments.

Reply:
The only reason I even know about this file is it showed up in the crates list on the left when I built locally. So it shows up in at least some circumstances! (Perhaps not on `docs.rs` though)

Reply:
TIL :)